src/plugins/filter/filter-fr.hbs

```diff
@@ -794,3 +795,52 @@
 		&lt;/ul&gt;
 	&lt;/div&gt;
 &lt;/div&gt;</code></pre>
+
+<h3 id="flt-customui">Interface persionnalisée</h3>
+
+<p>Personnalisez l'interface utilisateur générée par le plugiciel en ajoutant un élément et en définissant l'option <code>uiTemplate</code> à son sélecteur CSS.</p>
+
+<p>L'élément doit, au minimum, contenir un <code>&lt;input type="search"></code>.</p>
+
+<p>Pour afficher le nombre d'items visibles et le nombre total d'items, ajoutez un élément avec la classe <code>.wb-fltr-info</code>. Le plugiciel remplacera automatiquement "_NBITEM_" par le nombre d'éléments visibles et "_TOTAL_" par le total des éléments.</p>
+
+<div id="filterTemplate">
+	<div class="form-group">
+		<label for="customSearchUI">Rechercher</label>
+		<input type="search" class="form-control" placeholder="Tapez le nom d'une province..." id="customSearchUI">
+	</div>
+	<p class="wb-fltr-info">_NBITEM_ / _TOTAL_ provinces</p>
```

Comment:
The baseline requirement need to be document for when a custom search interface is used. Otherwise the author will only know by reverse engineering our working example and javascript.